$NetBSD: patch-arg,v 1.1 2013/08/23 11:31:29 wiz Exp $

Fix
src/util.c: In function 'ncprint':
src/util.c:172:3: error: 'va_list' undeclared (first use in this function)

--- src/util.c.orig	2013-01-18 09:12:16.000000000 +0000
+++ src/util.c
@@ -28,6 +28,7 @@
 #include <string.h>
 #include <stdlib.h>
 #include <ncurses.h>
+#include <stdarg.h>
 #include <locale.h>
 
 int winrows, wincols;
